site/_includes/docs-sidebar.html

```diff
@@ -5,39 +5,41 @@
   </button>
 </form>
 
-<nav class="collapse bd-links" id="bd-docs-nav" aria-label="Main navigation">
-  {%- assign page_slug = page.url | split: '/' | last -%}
-  {%- for group in site.data.nav -%}
-  {%- assign link = group.pages | first -%}
-  {%- assign link_slug = link.title | slugify -%}
-  {%- assign group_slug = group.title | slugify -%}
-  {%- assign active = nil -%}
+<div class="collapse d-lg-block" id="bd-docs-nav">
```

Comment:
Fix wrong breakpoint
```suggestion
<div class="collapse d-md-block" id="bd-docs-nav">
```